docs/consensus/overview.md

```diff
@@ -7,20 +7,75 @@ keywords:
     - Dilithium
 ---
 
-Subspace is powered by *Dilithium* - a lightweight and secure consensus mechanism that is environmentally friendly, permissionless, and fair. *Dilithium* is a Proof-of-Archival-Storage (PoAS), Nakamoto-style consensus protocol based on proofs of storing the blockchain's history. 
+Subspace is powered by *Dilithium* - a lightweight and secure consensus mechanism that is environmentally friendly, permissionless, and fair. *Dilithium* is a Proof-of-Archival-Storage (PoAS), Nakamoto-style consensus protocol based on proofs of storing the blockchain history. 
 
-In PoAS, farmers (not miners) store as many unique pieces of the blockchain history as their disk space allows. PoAS incentivizes the storage of the blockchain history, resolving the critical mechanism design failure, which has hindered scalability and led to centralization within Proof-of-Space blockchains like Filecoin and Chia. Since PoAS consensus is based on storage rather than computing power or wealth, it is eco-friendly and accessible to ordinary people with available disk space. This approach allows us to combine the high security of Bitcoin-style Proof-of-Work with the energy efficiency of Ethereum-style Proof-of-Stake. PoAS also provides the basis for a more generic solution to the blockchain bloat problem. 
+*Dilithium* is a second-generation PoAS consensus algorithm that uses erasure coding and KZG commitments for distributed archiving while combining polynomial encoding with an ASIC-resistant Proof-of-Space for plotting and AES-based Proof-of-Time. The protocol represents a significant step forward in security and user experience for Subspace Network participants. *Dilithium* is also designed to be SSD-friendly, further enhancing energy efficiency and decentralization. 
```

Comment:
Here we write that Dilithium is both PoS and PoT

Reply:
rephrased